Dockerfile

```diff
@@ -86,6 +86,12 @@ ENV PIP_DISABLE_PIP_VERSION_CHECK=true \
     PYTHONIOENCODING=utf8 \
     PYTHONUNBUFFERED=1
 
+
+# Update to the latest packages for the base image
```

Comment:
I would add a comment like "This allows to get CVE fixes ASAP, without waiting for new build of the base image
(see https://github.com/docker-library/python/issues/761 for an example of such an issue in the past
where the time between the CVE was discovered and the package update was X days, but the new base
image was updated only after Y days).

Reply:
Thanks!
It's addressed in the latest commit